doc/source/whatsnew/v2.1.0.rst

```diff
@@ -92,8 +92,8 @@ Other API changes
 
 Deprecations
 ~~~~~~~~~~~~
-- Deprecated accepting slices in :meth:`DataFrame.take`, call ``obj[slicer]`` or pass a sequence of integers instead (:issue:`51539`)
 - Deprecating pinning ``group.name`` to each group in :meth:`SeriesGroupBy.aggregate` aggregations; if your operation requires utilizing the groupby keys, iterate over the groupby object instead (:issue:`41090`)
+- Deprecated accepting slices in :meth:`DataFrame.take`, call ``obj[slicer]`` or pass a sequence of integers instead (:issue:`51539`)
```

Comment:
this shouldn't have changed, I presume something didn't go quite right when merging? could you restore it please?

Reply:
Hard reset before merge???